packages/core-mobile/app/services/bridge/UnifiedBridgeService.ts

```diff
@@ -93,73 +140,47 @@ export class UnifiedBridgeService {
       sourceChain: await this.buildChain(sourceNetwork)
     })
 
-    const fee = asset.address && feeMap[asset.address]
-
-    if (!fee) {
-      throw rpcErrors.invalidRequest({
-        data: {
-          reason: 'invalid fee'
-        }
-      })
+    if (isNativeAsset(asset)) {
+      // todo: handle this properly
+      return 0n
```

Comment:
what is the todo here? can you share more details?

Reply:
The getFee function has been re-implemented based on the extension implementation.